packages/mobile/locales/base/translation.json

```diff
@@ -1131,8 +1131,9 @@
     "delete": "Delete"
   },
   "syncingBankAccount": "Syncing bank account...",
-  "linkBankAccountSettingsTitle": "Link Bank Accounts",
+  "linkBankAccountSettingsTitle": "Linked Bank Accounts",
```

Comment:
shouldn't this title vary based on whether user has linked bank account yet? I would imagine it would be called `linked bank accounts` if there are bank accounts linked, while called `link bank accounts` if there are not bank accounts linked.

Reply:
The designs always call for `Linked bank accounts`. And i think its fine that it isn't a CTA since it also says `get access`